src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars--active"></div>
```

Comment:
Instead of using stars--active in every block you should to look at `nth child` pseudo-class https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child and write it in your css file. Because this code  difficult to maintain